docs/testing.rst

```diff
@@ -223,8 +223,8 @@ there does not seem to be a good way to do that, consider switching to
 application factories (see :ref:`app-factories`).
 
 Note however that if you are using a test request context, the
-:meth:`~flask.Flask.before_request` functions are not automatically called
-same for :meth:`~flask.Flask.after_request` functions.  However
+:meth:`~flask.Flask.before_request` functions are not automatically called.
+Same for the :meth:`~flask.Flask.after_request` functions.  However
```

Comment:
"If you are using a test request context, the before_request and after_request functions are not called automatically."